onnx/defs/generator/old.cc

```diff
@@ -12,6 +12,116 @@ This operator produces a constant tensor. Exactly one of the provided attributes
 or value_* must be specified.
 )DOC";
 
+void ConstantInference(InferenceContext& ctx) {
```

Comment:
Note: No change to this logic in the file in this PR. It just moves the type-and-shape-inference function for the Constant op into a separately defined function. It was hard to set breakpoints inside this inference function and step through it in a debugger when it is included in a macro (as before).

Reply:
Not relevant to this PR: might be worthy to make ConstantInference as a common function with options to handle different versions of Constant. Just like https://github.com/onnx/onnx/blob/a1e97a1ff257c7e3b1ecd62dc4872fa79059a275/onnx/defs/tensor/utils.cc#L150